doc/RecordingControl.xml

```diff
@@ -1684,8 +1787,420 @@ SetRecordingJobMode(JobToken, Active)
+            <row>
+              <entry>
+                <para>Key</para>
+              </entry>
+              <entry>
+                <para>Value</para>
+              </entry>
+              <entry>
+                <para>Format</para>
+              </entry>
+            </row>
+          </thead>
+          <tbody>
+            <row>
+              <entry>
+                <para><literal>Version</literal></para>
+              </entry>
+              <entry>
+                <para><literal>1.0</literal></para>
+              </entry>
+              <entry>
+                <para><literal>major "." minor</literal></para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para><literal>Source</literal></para>
+              </entry>
+              <entry>
+                <para>Identifier of the physical device</para>
+              </entry>
+              <entry>
+                <para>String</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para><literal>EventTopic</literal></para>
+              </entry>
+              <entry>
+                <para>Topic of the event that triggered the recording job</para>
+              </entry>
+              <entry>
+                <para>String</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para><literal>Duration</literal></para>
+              </entry>
+              <entry>
+                <para>Duration of the segment</para>
+              </entry>
+              <entry>
+                <para>ISO 8601-1</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para><literal>FullContentType</literal></para>
+              </entry>
+              <entry>
+                <para>Combined content type of all media types contained in the recording</para>
+              </entry>
+              <entry>
+                <para><literal>video/mp4</literal> content type with <literal>codecs</literal> parameter according to RFC 6381</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para><literal>VideoChannel</literal></para>
+              </entry>
+              <entry>
+                <para>Video channel identifier</para>
+              </entry>
+              <entry>
+                <para>String</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para><literal>VideoBitrate</literal></para>
+              </entry>
+              <entry>
+                <para>Video bitrate in kilobits per second</para>
+              </entry>
+              <entry>
+                <para>Integer</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para><literal>VideoWidth</literal></para>
+              </entry>
```

Comment:
do we need video codec & audio codec information in object attributes?

Reply:
I inserted this based on a discussion with Patrick. If a camera puts video and audio codec information on a video segment object, a VMS can only look at video segment objects and still know whether the recording contains an audio track or not and the codec of the audio track. The same is true when putting other audio-related attributes on video segment objects.